docs/modules/ROOT/pages/list-of-camel-quarkus-extensions.adoc

```diff
@@ -51,6 +51,9 @@ Number of Camel components: 25 in 21 JAR artifacts (0 deprecated)
 | xref:extensions/microprofile-metrics.adoc[MicroProfile Metrics] (camel-quarkus-microprofile-metrics) +
 `microprofile-metrics:metricType:metricName` | 0.2 | Camel metrics exposed with Eclipse MicroProfile Metrics
 
+| link:https://camel.apache.org/components/latest/netty-component.html[Netty] (camel-quarkus-netty) +
+`netty:protocol:host:port` | 0.2 | Socket level networking using TCP or UDP with the Netty 4.x library.
+
```

Comment:
`0.2` is not correct. Please add 

```
<firstVersion>0.4.0</firstVersion>
```

property to `extensions/netty/runtime/pom.xml`